### PR TITLE
Fix/details

### LINKED ISSUE #43: Adjust Details view
- [x] Shorten main body
- [x] Align the favoriting button w/ Discogs link button
- [x] Add `Tooltip` to Discogs button
- [x] Add back button
- [x] Style genres on details page

### PR DESCRIPTION
**Pull request by rdtho2525**
# Selector FE PR

## What does this merge do? Check all that apply.
- [ ] Adds a new feature
- [X] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## Summary of changes
- Shortens the main content window to the length that best fits the content itself
- Fixes the alignment of the links to Discogs and the favorite button
- Adds a Tooltip label to the Discogs link
- Adds a back button to the Details page
- Re-styles the genre elements to

## Notes for reviewer (any known bugs, areas of interest, etc.)
- No known bugs at this time

## Next Steps (Future Iterations)
- Remaining chores

### CODE REVIEW

File: src/Components/Details/Details.css [hunk 1]
```diff
@@ -13,22 +13,35 @@
 }
 
 .details_main {
-  height: 100vh;
+  max-height: 750px;
```
Comment (Max9545) [inline src/Components/Details/Details.css:16]: Should maybe use min or max as a default styling method in life?

File: src/Components/Details/Details.js [hunk 2]
```diff
@@ -40,33 +43,41 @@ const DetailsModal = ({ title, determineFav, toggleFav, isFavorite }) => {
...
-            <div className='genre-container' data-cy='genre-container'>
-              { displayGenres(data1.album.genres) }
+      <section className="details_main" data-cy="modal"> 
+        <Link to="/" className="details_back-button click">
+          <KeyboardReturnIcon className="details_return-arrow"/>
+            Back
+        </Link> 
+        <div className="details_content">
+          <article className="details_box left">
+            <img data-cy="album-cover" className="details_album-cover shadow" src={data1.album.coverImage} alt={`${data1.album.title} album cover`}/>
```
Comment (Max9545) [inline src/Components/Details/Details.js]: I think class names should come first and data-cy second

File: src/Components/Details/Details.css (hunk 1 above)
Comment (rdtho2525) [reply src/Components/Details/Details.css:16]: It's certainly come in handy the last few days!

File: src/Components/Details/Details.js (hunk 2 above)
Comment (rdtho2525) [reply src/Components/Details/Details.js]: Thanks for letting me know! This change has now been made.

File: src/Components/Details/Details.css [hunk 3]
```diff
@@ -128,25 +136,18 @@
...
-
-  
 }
 
 @media only screen and (min-width: 700px) {
-  .modal-container {
-    display: flex;
-    align-items: center;
+  .details_main {
+    width: 95vw;
```
Comment (monshan) [inline src/Components/Details/Details.css]: vw refers to the viewport so I would recommend the main be 95% of its parent, we can change the grid-area arrangement of the app if we want the details_main to cover more area

**Review by monshan — CHANGES_REQUESTED** [review]
Thanks for cleaning this up so much and getting the back button up and functioning, just suggesting one change to some css units and this will be good to go

File: src/Components/Details/Details.css (hunk 3 above)
Comment (rdtho2525) [reply src/Components/Details/Details.css]: This change is now made - thanks for the recommendation!

**Pull request merged**